pkl/builtins/zizmor.pkl

```diff
@@ -0,0 +1,90 @@
+import "../Builtins.pkl"
+import "../Config.pkl"
+import "./test/helpers.pkl"
+
+@Builtins.meta {
+  category = "Infrastructure"
+  description = "GitHub Actions security static analysis tool"
+}
+zizmor = new Config.Step {
+  glob =
+    List(
+      ".github/workflows/*.yml",
+      ".github/workflows/*.yaml",
+      ".github/dependabot.yml",
+      "**/action.yml",
+      "**/action.yaml",
+    )
```

Comment:
I referenced the values in the [official pre-commit configuration file for zizmorcore.](https://github.com/zizmorcore/zizmor-pre-commit/blob/120fe9ac3c965ab7decf67e8b9ca24b274230eed/.pre-commit-hooks.yaml#L6)